test/e2e/kubectl.go

```diff
@@ -1695,6 +1695,83 @@ metadata:
 			}
 		})
 	})
+
+	framework.KubeDescribe("Kubectl deployment reaper", func() {
+		var nsFlag string
+		var name string
+		var labelFlag string
+
+		BeforeEach(func() {
+			nsFlag = fmt.Sprintf("--namespace=%v", ns)
+			name = "e2e-test-nginx-deployment"
+			labelFlag = fmt.Sprintf("-l run=%v", name)
+		})
+
+		// No conformance label added as it is an explicit version skew test
```

Comment:
No need to mention skew test, because this is an e2e test. Skew test is something we run on top of it.